.github/workflows/main.yaml

```diff
@@ -7,56 +7,12 @@ on:
     branches: master
 
```

Comment:
With upstream dev test runs, it might be useful to add nightly test runs

```suggestion
 schedule:
    - cron: "0 0 * * *"
```

Thoughts?

Reply:
Let's give it a try!